build.gradle

```diff
@@ -21,8 +21,8 @@ apply plugin: 'com.palantir.docker'
 group = 'kingstonduo'
 
 bootJar {
-    baseName = 'cassiopeia-data'
-    version =  '1.2'
+    archiveBaseName = 'cassiopeia-data'
+    archiveVersion =  '1.2'
```

Comment:
Two questions on this:
Will the name change cause any downstream issues in code, or is this purely a tag mechanism in gradle?
Why is this marked as archived now?

Reply:
This, as well as the change below were needed because I needed to upgrade the version of gradle wrapper used. The build.gradle file is used to build the artifact and should not have any downstream impact. They renamed baseName to archiveBaseName and version to archiveVersion. I think they are using the term archive to mean artifact or output.